bin/resources/GameIndex.yaml

```diff
@@ -148,6 +159,15 @@ PBPX-95601:
 PCPX-96311:
   name: "Gran Turismo 3 Trial Disk Volume 1"
   region: "NTSC-J"
+PCPX-96322:
+  name: "ICO [Trial]"
+  region: "NTSC-J"
+  clampModes:
+    eeClampMode: 2  # Otherwise freezes in various spots, check full intro.
+    vuClampMode: 1  # Otherwise camera does not focus correctly on main character.
```

Comment:
remove double space before hash on both lines

Reply:
Now it should be fine.